### PR TITLE
Some refactoring

### PR DESCRIPTION
**Pull request by xepozz**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Tests pass?   | ✔️

### CODE REVIEW

File: src/Widget/LinkSorter.php [hunk 1]
```diff
@@ -2,21 +2,20 @@
...
  * LinkSorter will generate a hyperlink for every attribute declared in [[sort]].
- *
  * For more details and usage information on LinkSorter, see the [guide article on sorting](guide:output-sorting).
+ * @method static LinkSorter widget()
  */
 class LinkSorter extends Widget
 {
     /**
-     * @var Sort the sort definition
+     * @var \Yiisoft\Data\Reader\Sort the sort definition
```
Comment (samdark) [inline src/Widget/LinkSorter.php]: That's not related. Let's avoid the dependency like that.

File: composer.json [hunk 2]
```diff
@@ -25,11 +25,12 @@
         "psr/log": "^1.1",
         "yiisoft/aliases": "^3.0@dev",
         "yiisoft/arrays": "^3.0@dev",
+        "yiisoft/data": "^3.0@dev",
```
Comment (samdark) [inline composer.json]: I'd avoid this dependency. See the comment where it's used.

Comment (xepozz) [reply composer.json]: It needs for 
https://github.com/yiisoft/view/pull/58/commits/1043fa5004b2d251519e74d1e143fe4d92f05411#diff-a3592cd657d9165af5eacaef68d15024R24

Comment (samdark) [reply composer.json]: LinkPager should be moved to https://github.com/yiisoft/yii-dataview

Comment (xepozz) [reply composer.json]: And LinkSorter too?

Comment (samdark) [reply composer.json]: Yes.

**Comment by samdark** [comment]
Thanks!

**Pull request merged**